tests/pr_test.go

```diff
@@ -221,6 +221,7 @@ func TestRunRoksPattern(t *testing.T) {
 	}
```

Comment:
@ocofaigh @toddgiguere The `PostApplyHook` never executes because `TestRunRoksPattern` are skipped when enableSchematicsTests is True.  Where we should we add the hook because other test methods like TestRunVSIPattern, TestRunVpcPattern are also getting skipped.

Reply:
The landing zone tests executes in schematics and post apply hook is yet to be supported in schematics.

Reply:
Taking this to deep dive today to decide whether we add a support for post apply hook in schematics or is there any other way to approach it?